src/api/spec/features/webui/main_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "browser_helper"
+
+RSpec.feature "OBS main page", :type => :feature, :js => true do
+  let(:admin_user) { create(:admin_user) }
+  let(:user) { create(:confirmed_user) }
+
+  describe "#index" do
```

Comment:
The title should be more descriptive in this case (is not a controller spec, and supposedly we are describing a feature or part of it, not a method/action inside a controller)... I think that something like "main page", or just merge the describe and the context in only one thing.